lua-language-server/Dockerfile

```diff
@@ -0,0 +1,25 @@
+FROM alpine:3.13.5 as build
+
+ENV VERSION="1.20.5"
```

Comment:
I like that the VERSION is specified here -- it'd be very cool to copy this as a pattern.

Reply:
I actually had this in the `docker-langserver` and removed it to debug - will re-add.